rpcs3/Emu/RSX/VK/vkutils/device.cpp

```diff
@@ -73,20 +74,28 @@ namespace vk
 				features2.pNext               = &shader_barycentric_info;
 			}
 
+			if (device_extensions.is_supported(VK_EXT_CUSTOM_BORDER_COLOR_EXTENSION_NAME))
+			{
+				custom_border_color_info.sType = VK_STRUCTURE_TYPE_PHYSICAL_DEVICE_CUSTOM_BORDER_COLOR_FEATURES_EXT;
+				custom_border_color_info.pNext = features2.pNext;
+				features2.pNext                = &custom_border_color_info;
+			}
+
 			auto _vkGetPhysicalDeviceFeatures2KHR = reinterpret_cast<PFN_vkGetPhysicalDeviceFeatures2KHR>(vkGetInstanceProcAddr(parent, "vkGetPhysicalDeviceFeatures2KHR"));
 			ensure(_vkGetPhysicalDeviceFeatures2KHR); // "vkGetInstanceProcAddress failed to find entry point!"
 			_vkGetPhysicalDeviceFeatures2KHR(dev, &features2);
 
-			shader_types_support.allow_float64 = !!features2.features.shaderFloat64;
-			shader_types_support.allow_float16 = !!shader_support_info.shaderFloat16;
-			shader_types_support.allow_int8    = !!shader_support_info.shaderInt8;
-			framebuffer_loops_support          = !!fbo_loops_info.attachmentFeedbackLoopLayout;
-			barycoords_support                 = !!shader_barycentric_info.fragmentShaderBarycentric;
-			features                           = features2.features;
+			shader_types_support.allow_float64   = !!features2.features.shaderFloat64;
+			shader_types_support.allow_float16   = !!shader_support_info.shaderFloat16;
+			shader_types_support.allow_int8      = !!shader_support_info.shaderInt8;
```

Comment:
these 3 lines seem to be changed for no reason, at least they could be aligned better

Reply:
Fixed.